tools/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ if(ENABLE_PYTHON)
             COMPONENT python_tools_${PYTHON_VERSION})
 
     install(DIRECTORY benchmark/
```

Comment:
what if we replace `benchmark/` with `benchmark`?

Reply:
That seems to work.